rotkehlchen/tests/unit/test_ethereum_names.py

```diff
@@ -0,0 +1,72 @@
+from typing import Dict, List
+
+import pytest
+
+from rotkehlchen.chain.ethereum.names import NamePrioritizer
+from rotkehlchen.chain.ethereum.types import string_to_evm_address
+from rotkehlchen.types import DEFAULT_ADDRESS_NAME_PRIORITY, AddressNameSource, ChecksumEvmAddress
+
+
+class TestNameFetcher:
+    address_names: Dict[ChecksumEvmAddress, str]
+
+    def __init__(self, address_names: Dict[ChecksumEvmAddress, str]):
+        self.address_names = address_names
+
+    def get_addresses_names(self, _) \
+            -> Dict[ChecksumEvmAddress, str]:
+        return self.address_names
+
+
+@pytest.fixture(name='evm_address')
+def fixture_evm_address() -> ChecksumEvmAddress:
+    return string_to_evm_address("0xasdf")
+
+
+@pytest.fixture(name='name_source_ids')
+def fixture_name_source_ids() -> List[AddressNameSource]:
+    return DEFAULT_ADDRESS_NAME_PRIORITY
+
+
+@pytest.fixture(name='fetchers')
```

Comment:
The linter gave me errors about `Redefining name ... from outer scope (redefined-outer-name)`. This happens because of the fixtures defined in the same module: https://stackoverflow.com/questions/46089480/pytest-fixtures-redefining-name-from-outer-scope-pylint
I've found the mentioned fix in other places in the code, so I decided to do it that way.